static/js/src/advantage/subscribe/reducers/form-reducer.js

```diff
@@ -28,7 +31,15 @@ const prefixMap = {
   "infra+apps": "uaia",
 };
 
-const productsArray = Object.entries(window.productList);
+function hasMonthly(productID) {
+  let numberOfPeriods = 0;
+  productsArray.forEach((product) => {
+    if (product[1].productID === productID) {
+      numberOfPeriods++;
+    }
+  });
+  return numberOfPeriods > 1;
+}
```

Comment:
Can we be more precise with the logic? If they add daily this would return true. Here is an example of what I mean;

```js 
function hasMonthly(productID) {
  let containsMonthly = false;
  productsArray.forEach((product) => {
    if (product[1].productID === productID && product[1].period == "monthly") {
      containsMonthly = true;
    }
  });
  return containsMonthly;
}
```

Reply:
The naming of the function is maybe not right but if they add daily we would still want to display the billing period dropdown, right?

Reply:
I think the name is perfectly correct and clear for the functional logic it returns. If they add daily this will return true even if the options are ["yearly", "daily"].

Reply:
If we are in a rush, this is ok. But, it would be better to have logic that knows what it's displaying. 

I would like this function to return an array of periods for the selected product so we can check the length to hide the select and use the array to add items to the select box in a clear known way. If this is blocking the launch it would work but please revisit as this could come back to bite us in the future.

Reply:
You're right it's better this way I updated it